src/config.py

```
@@ -0,0 +1,243 @@
+        section = self.SECTION_MAP.get(prefix, prefix.lower())
+        toml_data = TOML_CONFIG.get(section, {})
+
+        # Convert keys to uppercase to match field names
+        return {key.upper(): value for key, value in toml_data.items()}
+
+
+class TomlSettings(BaseSettings):
+    """Base settings class that loads from TOML config first, then env vars."""
+
+    @classmethod
+    def settings_customise_sources(
```

Ruff (0.11.9)

Missing return type annotation for classmethod `settings_customise_sources`

(ANN206)